README.md

```diff
@@ -115,37 +115,25 @@ usage: saml2aws [<flags>] <command> [<args> ...]
 
 Flags:
-      --help                   Show context-sensitive help (also try --help-long
-                               and --help-man).
+      --help                   Show context-sensitive help (also try --help-long and --help-man).
       --version                Show application version.
       --verbose                Enable verbose logging
-  -i, --provider=PROVIDER      This flag is obsolete. See:
-                               https://github.com/Versent/saml2aws#configuring-idp-accounts
-  -a, --idp-account="default"  The name of the configured IDP account. (env:
-                               SAML2AWS_IDP_ACCOUNT)
+  -i, --provider=PROVIDER      This flag is obsolete. See: https://github.com/Versent/saml2aws#configuring-idp-accounts
+  -a, --idp-account="default"  The name of the configured IDP account. (env: SAML2AWS_IDP_ACCOUNT)
       --idp-provider=IDP-PROVIDER
-                               The configured IDP provider. (env:
-                               SAML2AWS_IDP_PROVIDER)
+                               The configured IDP provider. (env: SAML2AWS_IDP_PROVIDER)
       --mfa=MFA                The name of the mfa. (env: SAML2AWS_MFA)
-  -s, --skip-verify            Skip verification of server certificate.
-      --url=URL                The URL of the SAML IDP server used to login.
-                               (env: SAML2AWS_URL)
-      --username=USERNAME      The username used to login. (env:
-                               SAML2AWS_USERNAME)
-      --password=PASSWORD      The password used to login. (env:
-                               SAML2AWS_PASSWORD)
-      --mfa-token=MFA-TOKEN    The current MFA token (supported in Keycloak,
-                               ADFS, GoogleApps). (env: SAML2AWS_MFA_TOKEN)
-      --role=ROLE              The ARN of the role to assume. (env:
-                               SAML2AWS_ROLE)
-      --aws-urn=AWS-URN        The URN used by SAML when you login. (env:
-                               SAML2AWS_AWS_URN)
-      --duo-mfa-option         The MFA option you want to use to authenticate (env: SAML_DUO_MFA_OPTION)
+  -s, --skip-verify            Skip verification of server certificate. (env: SAML2AWS_SKIP_VERIFY)
+      --url=URL                The URL of the SAML IDP server used to login. (env: SAML2AWS_URL)
+      --username=USERNAME      The username used to login. (env: SAML2AWS_USERNAME)
+      --password=PASSWORD      The password used to login. (env: SAML2AWS_PASSWORD)
+      --mfa-token=MFA-TOKEN    The current MFA token (supported in Keycloak, ADFS, GoogleApps). (env: SAML2AWS_MFA_TOKEN)
+      --role=ROLE              The ARN of the role to assume. (env: SAML2AWS_ROLE)
+      --aws-urn=AWS-URN        The URN used by SAML when you login. (env: SAML2AWS_AWS_URN)
       --skip-prompt            Skip prompting for parameters during login.
-      --exec-profile           Execute the given command utilizing a specific profile from your ~/.aws/config file
```

Comment:
why did this get removed from the readme?

Reply:
I just ran the help command and updated the version in the README. May have been removed in a previous PR.

Reply:
https://github.com/Versent/saml2aws/pull/420 has some fixes for the issues you have identified.